ui/src/views/ThirdPartyLogin/index.jsx

```diff
@@ -102,7 +107,10 @@ export default class ThirdPartyLogin extends Component {
             formData={formData}
           />
         )}
-        {!user && <Homepage />}
+        {!user && isOneLoginStrategy && (
+          <Redirect to={`/login/${window.env.UI_LOGIN_STRATEGY_NAMES}`} />
```

Comment:
It should open in a new tab instead of redirecting, similar to the current implementation. Apologies for not making this clear.

https://github.com/taskcluster/taskcluster/blob/31e90eaa058c2d360e144c43a2745bc9bd0de064/ui/src/components/SignInDialog/index.jsx#L96

Reply:
Is this approach good?
{!user && isOneLoginStrategy && (
          window.open(`/login/${window.env.UI_LOGIN_STRATEGY_NAMES}`))}

Reply:
Probably. I think you can do `window.open(..., '_blank');`